Source/Core/InputCommon/InputConfig.h

```diff
@@ -31,6 +31,7 @@ class InputConfig
 	ControllerEmu* GetController(int index);
 	void ClearControllers();
 	bool ControllersNeedToBeCreated() const;
+	bool IsControllerControlledBySomeTypeOfGamepadDevice(int index) const;
```

Comment:
`IsControllerControlledByGamepadDevice` means the same thing in that instance.

> I put it there for a reason.

Sure, but it's also unnecessary